easybuild/framework/easyconfig/templates.py

```diff
@@ -299,6 +300,14 @@ def template_constant_dict(config, ignore=None, skip_lower=None, toolchain=None)
             # don't fail just because we couldn't resolve this template
             _log.warning("Failed to create mpi_cmd_prefix template, error was:\n%s", err)
 
+    # step 6. CUDA compute capabilities
+    cuda_compute_capabilities = build_option('cuda_compute_capabilities') or config.get('cuda_compute_capabilities')
```

Comment:
Maybe add a comment mentioning that the `--cuda-compute-capabilities` values overrides value of the `cuda_compute_capabilities` parameter in the easyconfig file?

Reply:
Does this suffice: https://github.com/easybuilders/easybuild-framework/pull/3382/commits/e0a64144bdab3c7b5230dcdfc07728a4360ede58

Reply:
yep!